src/Microsoft.Identity.Abstractions/DownstreamApi/IDownstreamApi.HttpMethods.tt

```diff
@@ -24,9 +28,20 @@ namespace Microsoft.Identity.Abstractions
     public partial interface IDownstreamApi
     {
 <#
+  foreach(string framework in new string[]{  "all", "net8" } )
+  {
+  
+    if (framework == "net8")
+    {
+#>
+
+#if NET8_0_OR_GREATER
+<#
+    }
+
   foreach(string httpMethod in new string[]{  "Get", "Post", "Put", "Patch", "Delete"} )
   {
-   if (httpMethod == "Patch")
+   if (httpMethod == "Patch" && framework != "net8")
```

Comment:
I think you still want these "Patch" methods on net8. We probably need to update line 48 to:

```
#if NETSTANDARD2_1_OR_GREATER || NET
```

or similar.